### DISCUSSION TITLE
Strimzi installation in ipv6

### DISCUSSION THREAD

**Discussion by Rabzou** [strimzi/strimzi-kafka-operator#7148] [Q&A] [answered]
Hello,
Is there a documented way to install the operator on a singleStack ipv6 cluster?
I'm getting this issue with all strimzi versions:
[https://github.com/strimzi/strimzi-kafka-operator/issues/4002](https://github.com/strimzi/strimzi-kafka-operator/issues/4002)

**Comment by scholzj** [comment, marked as answer]
I do not think OkHttp addressed this. So you will need to disable the hostname verification as described in the linked issue:
```
  - name: KUBERNETES_DISABLE_HOSTNAME_VERIFICATION
    value: "true"
```

**Reply by Rabzou** [reply]
Thanks for the reply!
